services/client/src/components/card/Card.tsx

```diff
@@ -45,7 +45,7 @@ const Card = ({ children, ...props }: CardProps) => {
             <DataContainer>
               <DataHeader key={`pool-${Date.now()}`}>Pool:</DataHeader>
               <DataValue key={`${cap}-${Date.now()}`}>
-                {transformFromWei(cap, tokenDecimals)} {tokenName}
+                {transformFromWei(cap, tokenDecimals)} {tokenName || 'TARA'}
```

Comment:
We should cover this fallback when the `tokenName` is getting its value assigned, I believe
```suggestion
                {transformFromWei(cap, tokenDecimals)} {tokenName}
```

Reply:
Yeah, but currently this might a different issue altogether. Might need to handle it in another PR after we decide on the corect approach.